WindowsServerDocs/failover-clustering/deploy-two-node-clustered-file-server.md

```diff
@@ -54,22 +54,22 @@ For a failover cluster in Windows Server 2016 or Windows Server 2019 to be consi
 
 - The fully configured solution (servers, network, and storage) must pass all tests in the validation wizard, which is part of the failover cluster snap-in.
 
-The following will be needed for a two-node failover cluster.
+The following is needed for a two-node failover cluster.
 
 - **Servers:** We recommend using matching computers with the same or similar components.  The servers for a two-node failover cluster must run the same version of Windows Server. They should also have the same software updates (patches).
 
 - **Network Adapters and cable:** The network hardware, like other components in the failover cluster solution, must be compatible with Windows Server 2016 or Windows Server 2019. If you use iSCSI, the network adapters must be dedicated to either network communication or iSCSI, not both. In the network infrastructure that connects your cluster nodes, avoid having single points of failure. There are multiple ways of accomplishing this. You can connect your cluster nodes by multiple, distinct networks. Alternatively, you can connect your cluster nodes with one network that is constructed with teamed network adapters, redundant switches, redundant routers, or similar hardware that removes single points of failure.
 
    > [!NOTE]
-   > If the cluster nodes are connected with a single network, the network will pass the redundancy requirement in the Validate a Configuration wizard.  However, the report will include a warning that the network should not have a single point of failure.
+   > If the cluster nodes are connected with a single network, the network passes the redundancy requirement in the Validate a Configuration wizard.  However, the report includes a warning that the network should not have a single point of failure.
 
 - **Device Controllers or appropriate adapters for storage:**
     - **Serial Attached SCSI or Fibre Channel:** If you are using Serial Attached SCSI or Fibre Channel, in all clustered servers, all components of the storage stack should be identical. It is required that the multipath I/O (MPIO) software and Device Specific Module (DSM) software components be identical.  It is recommended that the mass-storage device controllers—that is, the host bus adapter (HBA), HBA drivers, and HBA firmware—that are attached to cluster storage be identical. If you use dissimilar HBAs, you should verify with the storage vendor that you are following their supported or recommended configurations.
```

Comment:
This use of "be" looks archaic/poetic or even like some form of broken English:
 ```suggestion
    - **Serial Attached SCSI or Fibre Channel:** If you are using Serial Attached SCSI or Fibre Channel, in all clustered servers, all components of the storage stack should be identical. It is required that the multipath I/O (MPIO) software and Device Specific Module (DSM) software components are identical.  It is recommended that the mass-storage device controllers—that is, the host bus adapter (HBA), HBA drivers, and HBA firmware—that are attached to cluster storage be identical. If you use dissimilar HBAs, you should verify with the storage vendor that you are following their supported or recommended configurations.
```
Suggestion: [It is required that the multipath I/O (MPIO) software and Device Specific Module (DSM) software components **are** identical.]
(If you wanted to use "be", it could have been written as follows: [The requirement for the multipath I/O (MPIO) software and Device Specific Module (DSM) software components is that they **need to be** identical.] )

Reply:
BTW, the section I have made this suggestion for is part of the previously unchanged document.